drop-boxes/register-attachments-dropbox/register-attachment-dropbox.py

```diff
@@ -84,26 +84,14 @@ def process(transaction):
 	sa = transaction.getSampleForUpdate(sampleID)
 	space = sa.getSpace()
 	if not attachmentReady:
-		expID = '/' + space + '/' + project + '/'+ project+'_INFO'
-		exp = transaction.getExperimentForUpdate(expID)
-		if not exp:
-			exp = transaction.createNewExperiment(expID, "Q_PROJECT_DETAILS")
+		infoSampleID = "/"+space+"/"+code
+		sa = transaction.getSampleForUpdate(infoSampleID)
+	if not sa:
+		exp = transaction.createNewExperiment('/' + space + '/' + project + '/'+ project+'_INFO', "Q_PROJECT_DETAILS")
```

Comment:
why did you change this? Is this related to the documentation?

Reply:
Unfortunately not. The script is mainly used on another datastore server (that is not kept up to date with this repository, since it only contains this script...long story), so I wanted to make sure that the Github documentation is referencing the version currently used in production.

Reply:
I think it would be better to make this change in another PR to make changes more explicitly visible in the commit history. Can you imagine to create a small PR for that? I volunteer to review it fast ;)

Reply:
Good idea, I will do that!